### ISSUE TITLE
Incorrect response for Toolbar on/off switch

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#423]
I miss-understood what the on/off switch for the Toolbar does. It does not hide and show the Toolbar like the Show/Show Toolbar button, it turns the toolbar on and off.

I realize that should have been obvious, but some how I missed that detail.

The current responses for the Toolbar on/off switch are:
- Toolbar hidden.
- Toolbar shown.

They should be changed to something a little different. Maybe
- Toolbar off.
- Toolbar on.

Now that I think about this a bit more, I'm thinking that's a little vague. I will think about this over the weekend.

Other possible ideas:
- Toolbar disabled.
- Toolbar in sim disabled.

**Comment by terracoda** [comment]
Tagging the team in case anyone has any brilliant ideas about context response that clearly conveys that the toolbar that's in the sim is being turned off and on when the button is toggled. Keep in mind that in the future the toolbar will provide quick in-sim access to more features that just voicing.

Any thoughts, @emily-phet, @Matthew-Moore240, @BLFiedler, @jessegreenberg? I'll ask others on slack. 

We might just want to stick with the simplest option (Toolbar off./Toolbar on.)

**Comment by jessegreenberg** [comment]
Hmm, maybe something like
- Toolbar available.
- Toolbar removed.

?

**Comment by emily-phet** [comment]
@terracoda I like @jessegreenberg's suggestion above. That still leaves other options (like "toolbar open/closed" or "hidden/shown") for when the toolbar is just being collapsed/expanded.

Adding to those ideas - maybe "toolbar added/removed".

**Comment by terracoda** [comment]
I, too, like @jessegreenberg's idea, and I was also thinking added/removed would be good.

Do you think "to sim" is at all useful, like this:
- Toolbar removed. (when switched to off position)
- Toolbar added to sim. (when switched to on position)

It is a reminder the toolbar is actually in the sim, not in Preferences. And then when they traverse the sim, they will come across the Hide Toolbar and Show Toolbar and think, oh, this is the Toolbar I added.

Otherwise, I am good with the simpler version:
- Toolbar removed. 
- Toolbar added.

**Comment by emily-phet** [comment]
I like the addition of "to sim". I was wondering about issue as well...how will it be interpreted that it's not immediately available while you're still in the preference menu. I think this addition will help.

**Comment by terracoda** [comment]
Awesome, great team work.
@jessegreenberg we will go with our tri-combo solution:
- Toolbar removed.
- Toolbar added to sim.

**Comment by jessegreenberg** [comment]
OK sounds good. @terracoda  can you please review the responses in this version?

https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-preferences.15/phet/john-travoltage_en_phet.html

**Comment by terracoda** [comment]
Sounds perfect! Thanks @jessegreenberg!
Closing.
